containers/nginx-storage/crontab.txt

```diff
@@ -1,2 +1,2 @@
 # min   hour    day     month   weekday command
-0       2       *       *       *       /etc/crontabs/delete-old-files.sh > /dev/stdout
+0       2       *       *       *       /etc/crontabs/delete-old-files.sh 2> /dev/stderr > /dev/stdout
```

Comment:
why not just? 
```bash
/etc/crontabs/delete-old-files.sh > /dev/stdout  2>&1
```
?

Reply:
Because it redirects `stderr` to `stdout`.